src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -27,6 +89,51 @@ class FilterPopup extends React.Component {
     });
   }
 
+  componentWillReceiveProps(nextProps) {
+    this.setState({ selectedFilters: nextProps.selectedFilters });
+  }
+
+  onFilterClick(filterId, filter) {
+    const selectedFilters = this.state.selectedFilters;
+
+    if (filter.selected) {
+      selectedFilters[filterId].push(filter);
```

Comment:
Sometimes when I click around the filter on my local or dev, I will saw this error on the browser console:
`uncaught TypeError: Cannot read property 'push' of undefined` and the code indicates to this line.

![screen shot 2017-09-15 at 4 07 49 pm](https://user-images.githubusercontent.com/3506922/30501368-34a67a74-9a30-11e7-9c30-b51c05f9489d.png)

Not sure what caused it but probably sometimes `selectedFilters[filterId]` is not an array?

Reply:
Yup, fixing it in my next PR.

Reply:
:+1: